actions/signing-event/action.yml

```diff
@@ -59,60 +59,83 @@ runs:
         cat status-output >> "$GITHUB_STEP_SUMMARY"
       shell: bash
 
-    - id: file-issue
+    - id: update-pr
       uses: actions/github-script@60a0d83039c74a4aee543508d2ffcb1c3799cdea
       env:
         STATUS: ${{ steps.status.outputs.status }}
       with:
         github-token: ${{ inputs.token }}
         script: |
           const fs = require('fs')
-          message = fs.readFileSync('./status-output').toString()
-          summary = ''
-
-          issue = 0
-          const repo = context.repo.owner + "/" + context.repo.repo
-          const issues = await github.rest.issues.listForRepo({
-            owner: context.repo.owner,
-            repo: context.repo.repo,
-            labels: [process.env.GITHUB_REF_NAME],
+          const title = `Signing event: ${process.env.GITHUB_REF_NAME}`
+          const repo = `${context.repo.owner}/${context.repo.repo}`
+          const prs = await github.rest.search.issuesAndPullRequests({
+            q: `in:title+"${title}"+state:open+type:pr+repo:${repo}`
```

Comment:
* uses `rest.search` because `rest.issues` and `rest.pulls` do not seem to allow filtering pull requests with anything useful.
* uses title instead of label because `rest.pulls.create()` does not support labels. Using the title should be fine: it includes the branch name which we expect to be unique anyway